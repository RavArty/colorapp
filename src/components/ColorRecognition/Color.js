import React from 'react';
import { Circle } from 'react-shapes'

class Color extends React.Component {

  render(){
    
    return (
    <div >
      <div className = 'mw8 pa2 center br4 shadow-3'>
        {`${this.props.colorCode}, ${Number(this.props.value * 100).toFixed(2)}%  `}
        &nbsp;<Circle r={10} fill={{color: this.props.colorCode}} />
        
      </div>
      <div className='pb2'></div>
    </div>
  );
  }
  
}

export default Color;