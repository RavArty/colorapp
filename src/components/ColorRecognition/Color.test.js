import React from 'react'
import { shallow } from 'enzyme';
import Color from './Color'

it('expect to render Color component', () => {
  expect(shallow(<Color/>)).toMatchSnapshot()
})
