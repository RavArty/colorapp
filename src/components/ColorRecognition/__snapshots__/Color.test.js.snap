// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Color component 1`] = `ShallowWrapper {}`;
